superbench/runner/runner.py

```diff
@@ -233,6 +235,18 @@ def fetch_results(self):    # pragma: no cover
             )
         )
 
+    def __signal_handler(self, signum, frame):
+        """Signal handler for runner.
+
+        Args:
+            signum (int): Signal number.
+            frame (FrameType): Timeout frame.
+        """
+        if signum == signal.SIGINT or signum == signal.SIGTERM:
```

Comment:
I remember that there also has one signal named `SIGKILL`, do we need to catch it?

Reply:
It looks like we cannot catch it from https://docs.python.org/3/library/signal.html